README.md

```diff
@@ -33,6 +33,29 @@ make new adapter=my-adapter-name
 
 _If on a Mac, this requires `gnu-sed` to be installed and set as the default for the command `sed`._
 
+### Input
+
+This is an example, a JSON body the adapter will receive when plugged into the Chainlink node pipeline:
```

Comment:
```suggestion
When the Chainlink node sends a request to the external adapter, it will send a JSON body with the job run id as "id" and the request parameters as "data". Aggregator contracts of version 3 or above (FluxMonitor and Offchain Reporting) will also include the `RoundState` as the "meta" field in the request, serialized to a JSON object with lower camelCase keys.

An example request body can look like this:
```